packages/flutter/lib/src/material/drawer.dart

```diff
@@ -275,14 +275,21 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
                     child: new Focus(
                       key: _drawerKey,
                       child: config.child
-                    )
-                  )
-                )
-              )
-            ]
-          )
-        )
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
       );
     }
   }
+  @override
+  Widget build(BuildContext context) {
+    return new ListTileTheme(
+      style: ListTileStyle.drawer,
+      child: _buildDrawer(context),
```

Comment:
I did this because _buildDrawer(), the original build method body, has two very different returns, depending on if the drawer is dismissed. I was avoiding  duplicating the new ListTileTheme expression in both branches, to make it clear that it was the same for either drawer variation.